VndbSharp/Structs/Models/DatabaseStats/DatabaseStats.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace VndbSharp.Structs.Models.DatabaseStats
+{
+    [JsonObject]
```

Comment:
If i was correct in that it does not do what you expect, and you decide to remove it, go ahead and remove other instances of it as well please :)